src/python/CMakeLists.txt

```diff
@@ -26,6 +26,11 @@
 
 cmake_minimum_required(VERSION 3.18)
 
+# [DLIS-XXXX] TODO
+if(NOT (${TRITON_ENABLE_HTTP} AND ${TRITON_ENABLE_GRPC}))
+  return()
```

Comment:
Also print a message that the python package build is being skipped for now when the relevant frontends are enabled.